.travis.yml

```diff
@@ -28,6 +28,6 @@ before_install:
 - echo "password = $TX_PASSWD" >> ~/.transifexrc
 script:
 - yarn test
-- yarn run build
+- yarn build:browser
```

Comment:
Are we sure commands can be run w/o the `run` command, as a shortcut? Yarn doc doesn't mention it…